Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -0,0 +1,258 @@
+                }
+            }
+
+            return maxBalance;
+        }
+
+        private void SetNames(ref StringBuilder name, ref decimal maxBalance, string[] peopleBalances, decimal balance)
+        {
+            if (balance >= maxBalance)
+            {
+                if (balance > maxBalance)
+                {
+                    name = new StringBuilder();
+                }
+
+                if (name.Length != 0)
+                {
+                    name.Append(", ");
+                }
+
+                name.Append(peopleBalances[0]); //Builds list of names with highest balance
+                maxBalance = balance;
+            }
+        }
+
+        /// <summary>
+        /// Return name and loss of a person with a biggest loss (balance change negative).
+        /// </summary>
+        public string FindPersonWithBiggestLoss(string[] peopleAndBalances)
+        {
+            if (IsArrayNullOrEmpty(peopleAndBalances))
+            {
+                return NotAvailable;
+            }
+
+            var biggestLossEver = decimal.MaxValue;
+            var name = "";
+
+            for (var i = 0; i < peopleAndBalances.Length; i++)
+            {
+                var peopleBalances = peopleAndBalances[i].Split(",");
+
+                if (peopleBalances.Length <= 2)
+                {
+                    continue;
+                }
+
+                var biggestLossForPerson = GetBiggestLossForPerson(peopleBalances);
+
+                if (biggestLossForPerson < biggestLossEver)
+                {
+                    biggestLossEver = biggestLossForPerson;
+                    name = peopleBalances[0];
+                }
+            }
+
+            if (name == "")
+            {
+                return NotAvailable;
+            }
+
+            CultureInfo.CurrentCulture = CultureInfo.InvariantCulture;
+
+            var biggestLossEverNoComma = RemoveComma($"{biggestLossEver:C0}").Replace("(", "-").Replace(")", "");
+            return $"{name} lost the most money. {biggestLossEverNoComma}.";
+        }
+
+        private decimal GetBiggestLossForPerson(string[] peopleBalances)
+        {
+            var biggestLoss = decimal.MaxValue;
+
+            for (var i = 1; i < peopleBalances.Length; i++)
+            {
+                for (var j = i + 1; j < peopleBalances.Length; j++)
+                {
+                    var isBeforeBalanceValid = decimal.TryParse(peopleBalances[i], out var beginAmount);
+                    var isAfterBalanceValid = decimal.TryParse(peopleBalances[j], out var endAmount);
+
+                    if (isBeforeBalanceValid && isAfterBalanceValid)
+                    {
+                        biggestLoss = GetBiggestLoss(biggestLoss, beginAmount, endAmount);
+                    }
+                }
+            }
+
+            return biggestLoss;
+        }
+
+        private decimal GetBiggestLoss(decimal biggestLoss, decimal beginAmount, decimal endAmount)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make 'GetBiggestLoss' a static method.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5063788)